### ISSUE TITLE
[FR] Rename the default branch from `master` to `main`

### ISSUE THREAD

**Issue by apreshill** [rstudio/rmarkdown#2181]
https://github.com/github/renaming

See also:
+ https://github.com/rstudio/bookdown/issues/1198
+ https://github.com/rstudio/distill/issues/390

**Comment by yihui** [comment]
Done. Team members will need to rename the branch accordingly locally:

```
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```

**Comment by cderv** [comment]
@yihui Is this possible that the way you are renaming is closing some issues : see https://github.com/rstudio/rmarkdown/pull/1442 and other I believe could have been closed because `rstudio:master` was deleted

There is a renaming feature in Github I wanted to show you that I think is designed to keep PR and other references : https://github.com/github/renaming#renaming-existing-branches

It does not seem you are you using that ?

**Comment by yihui** [comment]
I was using the renaming feature you mentioned above. I don't know why the PR #1442 was closed. Sounds like a bug of Github?

**Comment by cderv** [comment]
That is weird. I don't think it is the only that closed. I got several notification 🤔

I'll check them and see if any needs to be reopened.

**Comment by cderv** [comment]
Ok the other issue closed was #1030 and I think #1583

There where closed it seems because the branch the PR was from does not exist anymore. 

Anyway, nothing that we won't deal with in the future. For bookdown, no closed issue following renaming so I am not sure why...
